code/game/objects/items/plushies.dm

```diff
@@ -236,6 +239,11 @@
 	icon_state = "teddy"
 
 //This one is only available with a pomfcoin
+/obj/item/toy/plushie/chicken/pomf
+	name = "plush pomf chicken"
+	desc = "An extremely soft and plushy chicken. Cluck!"
+	hug_reagents = list(DOCTORSDELIGHT,PARACETAMOL)
```

Comment:
Justify why hugging this plushie should be an infinite source of healing chemicals.

Reply:
Because pomf coin is a rare and unique in-game item that justifies being tradeable for another rare and unique in-game item.

Reply:
Rarity alone should make it valuable. I don't see how this is good for the game, nor how it makes sense at all.
Plushies *don't even have in-hands*, it's impossible to tell at a glance if someone is holding something that will let them regenerate infinitely with tricord-lite (doctor's delight).

Reply:
Well, too bad, this is how the original PR was like and Pomf approved of it. I don't recall you making this an issue back then, so it isn't going to suddenly become an issue now.
Doctor's Delight is not even that strong, it doesn't heal anything else besides main damage types (blood, clone, brain, organs, bones, etc) and is only injected 1 unit per hug.